addons/phone_validation/views/res_partner_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <record id="view_res_partner_filter" model="ir.ui.view">
+            <field name="name">res.partner.select.inherit.phone.validation</field>
```

Comment:
Base view is probably not correctly named, but here when adding new views you should follow guidelines: name should be res.partner.view.form.inherit.sms (if moved in SMS app) and its ID res_partner_view_form .

Reply:
done.